src/shared/components/GenericList/GenericList.js

```diff
@@ -210,55 +211,54 @@ export const GenericList = ({
     ));
   };
 
-  const tableClassNames = classnames(
-    'fd-table',
-    'fd-table--no-horizontal-borders',
-    { compact },
-  );
   const panelClassNames = classnames(
-    'generic-list',
+    'ui5-generic-list',
     {
       'fd-margin--md': !className?.includes('fd-margin'),
     },
     className,
   );
 
+  const tableClassNames = classnames('ui5-generic-list', {
+    'ui5-content-density-compact': compact,
+  });
+
   return (
-    <LayoutPanel className={panelClassNames} data-testid={testid}>
-      <LayoutPanel.Header className="fd-has-padding-left-small fd-has-padding-right-small">
-        <LayoutPanel.Head title={title} />
-        <LayoutPanel.Actions>{headerActions}</LayoutPanel.Actions>
-      </LayoutPanel.Header>
+    <>
+      <LayoutPanel className={panelClassNames} data-testid={testid}>
+        <LayoutPanel.Header className="fd-has-padding-left-small fd-has-padding-right-small">
+          <LayoutPanel.Head title={title} />
+          <LayoutPanel.Actions>{headerActions}</LayoutPanel.Actions>
+        </LayoutPanel.Header>
 
-      <LayoutPanel.Body className="fd-has-padding-none">
-        <table className={tableClassNames}>
-          {showHeader && (
-            <thead className="fd-table__header">
-              <tr className="fd-table__row">
-                <HeaderRenderer
-                  entries={entries}
-                  actions={actions}
-                  headerRenderer={headerRenderer}
-                />
-              </tr>
-            </thead>
+        <LayoutPanel.Body className="fd-has-padding-none">
+          <Table
+            className={tableClassNames}
+            columns={
+              <HeaderRenderer
+                entries={entries}
+                actions={actions}
+                headerRenderer={headerRenderer}
+              />
+            }
+          >
+            {renderTableBody()}
+          </Table>
+        </LayoutPanel.Body>
+        {!!pagination &&
+          (!pagination.autoHide ||
+            filteredEntries.length > pagination.itemsPerPage) && (
+            <LayoutPanel.Footer>
+              <Pagination
+                itemsTotal={filteredEntries.length}
+                currentPage={currentPage}
+                itemsPerPage={pagination.itemsPerPage}
+                onChangePage={setCurrentPage}
+              />
+            </LayoutPanel.Footer>
           )}
-          <tbody className="fd-table__body">{renderTableBody()}</tbody>
-        </table>
-      </LayoutPanel.Body>
-      {!!pagination &&
-        (!pagination.autoHide ||
-          filteredEntries.length > pagination.itemsPerPage) && (
-          <LayoutPanel.Footer>
-            <Pagination
-              itemsTotal={filteredEntries.length}
-              currentPage={currentPage}
-              itemsPerPage={pagination.itemsPerPage}
-              onChangePage={setCurrentPage}
-            />
-          </LayoutPanel.Footer>
-        )}
-    </LayoutPanel>
+      </LayoutPanel>
+    </>
```

Comment:
Please remove that, it will cause more conflicts with ui5-panel PR and doesn't need to be here.
```suggestion
```